### PR TITLE
Issue 3318 - Viewer facelift - Safetibase Treatments card

### LINKED ISSUE #3318: Safeti base - Treatment
### Description
Safeti base's UI is broken  in the viewer   in v5 - we need to update it to make it usable for users. This is for treatment

### Goals
- [ ]  As a user I want to use the treatment functionality from safetibase in v5

### Tasks
- [ ] Fix the inputs change to the new style
- [ ] The Suggest button is too close to the section above - move all fields down to create space.

### Related Resources
image

[MockUp](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=136550%3A176546)

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3318

#### Description
- Fixed styling in the main tab
- Partly fixed suggested-treatment modal styling (mockup is missing)

### CODE REVIEW

**Review by The-Daniel — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/v4Adapter/overrides/cards/safetiBase/treatments.overrides.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+					margin-top: 0;
+					height: 24px;
+					border: 1px solid ${({ theme }) => theme.palette.base.lightest};
+				}
+				
+				& ~ * {
+					position: relative;
+					border-radius: 4px;
+					background: #fff;
+					height: 23px;
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/v4Adapter/overrides/cards/safetiBase/treatments.overrides.ts]: yes, you are right. It was meant to be min-height instead. I fixed it

**Review by The-Daniel — APPROVED** [review]

**Comment by george-hadfield** [comment]
@Amantini1997 @carmenfan can we change the case of UNSET to Title case? Seems a bit shouty..

image

**Comment by carmenfan** [comment]
> > @Amantini1997 @carmenfan can we change the case of UNSET to Title case? Seems a bit shouty..
> > image
> 
> If @carmenfan is ok, I will proceed to fix that

@mosment3DRepo

**Comment by mosment3DRepo** [comment]
> > > @Amantini1997 @carmenfan can we change the case of UNSET to Title case? Seems a bit shouty..
> > > image
> > 
> > 
> > If @carmenfan is ok, I will proceed to fix that
> 
> @mosment3DRepo

Agreed to having it proper case. Can't find any evidence of why it was all caps in the first place

File: frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			font-weight: 400;
+			font-size: 13px;
+			line-height: 1.125rem;
+			color: ${({ theme }) => theme.palette.base.main};
+		}
+
+		.MuiInputBase-root {
+			svg {
+				/* TODO: fix after new palette is released */
+				color: #C1C8D5;
```
[the remark this answers is not in the document]
Comment (george-hadfield) [reply frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts:47]: Which part of the design is using #C1C8D5?

[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts:47]: @george-hadfield several components are, for instance, the dropdown border as well as the chevron icon

[the remark this answers is not in the document]
Comment (george-hadfield) [reply frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts:47]: @Amantini1997 apologies - can you make sure it matches the same colours as the other components.

**Review by george-hadfield — APPROVED** [review]

File: frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/v4Adapter/dialogs/suggestedTreatmentsDialog.overrides.ts:47]: > Move "this field is required" text for Safetibase title to the right
> 
> image

@george-hadfield That same layout is used for `Issues` and `Groups` Cards as well. 
image
Should I change them as well?

**Comment by Amantini1997** [comment]
> Move "this field is required" text for Safetibase title to the right
> 
> image

@george-hadfield That same layout is used in `Issues` and `Groups` Cards as well. 
image
Should I change them as well?

**Comment by george-hadfield** [comment]
> > Move "this field is required" text for Safetibase title to the right
> > image
> 
> @george-hadfield That same layout is used in `Issues` and `Groups` Cards as well. image Should I change them as well?

Yes please

**Review by sanmont3drepo — APPROVED** [review]

**Review by The-Daniel — APPROVED** [review]

**Pull request merged**